src/components/ListItem.jsx

```diff
@@ -1,5 +1,9 @@
 import './ListItem.css';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({ name, index }) {
+	return (
+		<li className="ListItem" key={index}>
```

Comment:
Does the li element need a key prop if there is already a key prop on the ListItem component rendered in List.jsx?